doc/content/devices/class-b/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Class B"
+description: ""
+---
+
+Class B end devices listen for downlink messages during ping slots. This allows applications to send messages to devices at predefined time slots, rather than waiting for a Class A uplink. When combined with multicast groups, this allows applications to send periodic downlinks to many devices at the same time.
+
+<!--more-->
+
+## Enabling and Disabling Class B
+
+In order to send Class B downlink messages to a single device, enable Class B support for the end device when creating or updating it with the `--supports-class-b` flag.
+
+For example, when enabling Class B for an existing device:
+
+```bash
+$ ttn-lw-cli end-devices set app1 dev1 --supports-class-b
+```
+
+This will enable the Class B downlink scheduling of the device. Downlink messages are now scheduled during the next available ping slot.
+
+To disable Class B scheduling, reset with `--supports-class-b=false`.
```

Comment:
@rvolosatovs please advise here. I believe this needs to be set via MAC settings; `supports_class_b` is mostly in indicator that the device can support class B.

Reply:
No, class B operation starts when sufficient data is provided and this is set to `true`
FYI: we wrote this section together with @benolayinka